packages/core/src/oidc/init.ts

```diff
@@ -114,6 +114,19 @@ export default async function initOidc(app: Koa): Promise<Provider> {
         return refreshTokenTtl ?? defaultRefreshTokenTtl;
       },
     },
+    extraTokenClaims: async (ctx, token) => {
```

Comment:
Where is this used? I didn't find a reference

Reply:
oidc internal method